exchangelib/properties.py

```diff
@@ -172,7 +172,7 @@ def attribute_fields(cls):
     @classmethod
     def supported_fields(cls, version=None):
         # Return non-ID field names. If version is specified, only return the fields supported by this version
```

Comment:
nit: "Return non-ID field names" -> "Return non-ID field names that aren't only used for syncback"?

Reply:
Will do!